CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+# Contributing
+
+Thank you for your interest in contributing to this project! 
+
+Please review the following guidelines before making your contribution.
+
+> [!IMPORTANT]
+> If you plan on making a significant contribution, we recommend that you first create a discussion describing your proposed contribution to the project. This allows the project maintainers to provide early feedback that can help guide your contribution.
+
+## Project setup
+
+Ensure that you have the following tools installed on your machine:
+
+- Java development kit (JDK) 17
+- IntelliJ IDEA Ultimate Edition
+- git-cliff (_maintainers only_)
+
+1. Fork the repository and clone it to your local machine.
+   ```shell
+   gh repo fork biomejs/biome-intellij --clone
+   ```
+
+## Development
+
+The plugin can be started in IDEA by running the `runIde` Gradle task. This will start a new instance of IDEA with the plugin loaded.
+
+```shell
+./gradlew runIde
+```
+
+### Running tests
+
+TO run the plugin's tests, run the `test` Gradle task.
```

Comment:
```suggestion
To run the plugin's tests, run the `test` Gradle task.
```

These tests are UI testing and before running `./gradlew test` the contributor needs to run `./gradlew runIdeForUiTests &` to open the IDE

Can you also add instructions to do so?

Reply:
Thanks, I've updated it.